src/main/java/com/hibernate/model/CinemaHall.java

```diff
@@ -0,0 +1,31 @@
+package com.hibernate.model;
+
+public class CinemaHall {
```

Comment:
this class shouldn't be in the PR

you car remove all redundant files from git control using the command:  `git rm --cached filename`
after that, you should commit all changes and push them to the PR. 

Then the files will stay locally in the project, but they will not be in PR